Crash seen with 0.6.4

This happens because some VQL function is returning an array (not a slice). Not sure which is doing that though...

```
panic: reflect: call of reflect.Value.IsNil on array Value

goroutine 3526 [running]:
reflect.Value.IsNil(...)
	/home/mic/go/src/reflect/value.go:1506
www.velocidex.com/golang/vfilter/utils.IsNil({0x1b6e6a0, 0xc002519440})
	/home/mic/go/pkg/mod/www.velocidex.com/golang/vfilter@v0.0.0-20220330071711-a02abef46c06/utils/utils.go:61 +0x1b2
www.velocidex.com/golang/vfilter.(*_SymbolRef).callFunction(0xc00278f900, {0x1ff3ee0?, 0xc001ebe180}, {0x2002d68?, 0xc0021ffdd0}, {0x1fea790?, 0x30cc830})
	/home/mic/go/pkg/mod/www.velocidex.com/golang/vfilter@v0.0.0-20220330071711-a02abef46c06/vfilter.go:1966 +0x476
www.velocidex.com/golang/vfilter.(*_SymbolRef).Reduce(0xc00278f900, {0x1ff3ee0, 0xc001ebe180}, {0x2002d68, 0xc0021ffdd0})
	/home/mic/go/pkg/mod/www.velocidex.com/golang/vfilter@v0.0.0-20220330071711-a02abef46c06/vfilter.go:1803 +0x1c6
www.velocidex.com/golang/vfilter.(*_Value).Reduce(0xc002282720, {0x1ff3ee0, 0xc001ebe180}, {0x2002d68, 0xc0021ffdd0})
```

--- a/artifacts/definitions/Windows/Detection/CryptnetUrlCache.yaml
+++ b/artifacts/definitions/Windows/Detection/CryptnetUrlCache.yaml
@@ -27,7 +27,8 @@ parameters:
       C:\Users\*\AppData\LocalLow\Microsoft\CryptnetUrlCache\**
   - name: SusSize
     description: "Size in bytes for CryptnetUrlCache content to be suspicious"
-    default: 10000
+    default: '10000'
+    type: int
   - name: HeaderRegex
     description: 'Regex of content headers.'
     default: '.'

--- a/artifacts/definitions/Windows/Detection/Thumbdrives/List.yaml
+++ b/artifacts/definitions/Windows/Detection/Thumbdrives/List.yaml
@@ -17,6 +17,7 @@ type: CLIENT_EVENT
 
 parameters:
   - name: maxDriveSize
+    type: int
     description: We ignore removable drives larger than this size in bytes.
     default: "32000000000"
 

--- a/artifacts/definitions/Windows/Detection/Yara/NTFS.yaml
+++ b/artifacts/definitions/Windows/Detection/Yara/NTFS.yaml
@@ -35,7 +35,9 @@ parameters:
     description: "Target drive. Default is a C:"
     default: "C:"
   - name: SizeMax
+    type: int
   - name: SizeMin
+    type: int
   - name: AllDrives
     type: bool
   - name: UploadHits

--- a/go.mod
+++ b/go.mod
@@ -121,12 +121,12 @@ require (
 	www.velocidex.com/golang/go-prefetch v0.0.0-20200722101157-37e4751dd5ca
 	www.velocidex.com/golang/oleparse v0.0.0-20211013063943-0334d69593c1
 	www.velocidex.com/golang/regparser v0.0.0-20190625082115-b02dc43c2500
-	www.velocidex.com/golang/vfilter v0.0.0-20220330071711-a02abef46c06
+	www.velocidex.com/golang/vfilter v0.0.0-20220428152704-2c10e2e5806d
 )
 
 require (
 	github.com/Velocidex/file-rotatelogs v0.0.0-20211221020724-d12e4dae4e11
-	github.com/Velocidex/ordereddict v0.0.0-20220411103415-79032cf99b1d
+	github.com/Velocidex/ordereddict v0.0.0-20220428153415-da46091cd216
 	github.com/evanphx/json-patch/v5 v5.6.0
 	github.com/shirou/gopsutil/v3 v3.21.11
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0
@@ -198,6 +198,7 @@ require (
 // replace www.velocidex.com/golang/evtx => /home/mic/projects/evtx
 // replace www.velocidex.com/golang/go-ese => /home/mic/projects/go-ese
 // replace github.com/Velocidex/ordereddict => /home/mic/projects/ordereddict
+
 // replace github.com/Velocidex/yaml/v2 => /home/mic/projects/yaml
 // replace www.velocidex.com/golang/go-prefetch => /home/mic/projects/go-prefetch
 // replace github.com/Velocidex/go-magic => /home/mic/projects/go-magic

